plugins/mcreator-localization/help/fr_FR/rangeditem/bullet_ignite_fire.md

```diff
@@ -1,3 +1,3 @@
-Check this parameter if you want it to light a fire when the ranged item hits a block.
+Cochez ce paramètre si vous voulez qu'il allume un feu lorsque l'objet à distance frappe un bloc.
 
-Note: this does not catch mobs on fire
\ No newline at end of file
+Remarque: cela n'attrape pas les mobs en feu
```

Comment:
Wrong translation again...
It should be "Remarque: cela ne met pas les mobs en feu.